roles/confluent.kafka_broker/templates/client.properties.j2

```diff
@@ -4,5 +4,3 @@
 {% for key, value in kafka_broker_client_properties|dictsort%}
 {{key}}={{value}}
 {% endfor %}
-default.api.timeout.ms=20000
```

Comment:
hey @mansisinha ,
I saw hardcoded timeout configs in kafka controller client properties as well https://github.com/confluentinc/cp-ansible/blob/master/roles/kafka_controller/templates/client.properties.j2 , we might need to remove it from there as well in separate PR, please check

Also, not sure if we are verifying any of the client props in molecule verify step, if yes, will it make sense to verify default timeout value or custom timeout value defined in `kafka_broker_custom_client_properties`?

Reply:
Yes, we need a separate PR for controller. For your second question, we are not defining this properties anywhere via our code, we reply on CP for the default values. For context, these configs were added in this [PR](https://github.com/confluentinc/cp-ansible/commit/3974a76b68f6e96ce89832164d53c091a455f715) because the default values are larger and we wanted to get the api results faster. However, based on the latest RCCA it makes more sense to use the default and give the customer the option to modify when needed. Does this sound good?

Reply:
do you mean to verify if these configs defined in custom_client_properties are getting passed on to client.properties file or not?